assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -320,7 +321,7 @@
         @include breakpoint("medium") {
             border-left: 1px solid stencilColor("navPages-subMenu-separatorColor");
             display: inherit;
-            height: $navPage-subMenu-horizontal-height;
```

Comment:
Instead of removing this variable please change the value of the variable to `auto`, in `assets/scss/settings/stencil/navPages/_settings.scss`